devo/api/client.py

```diff
@@ -852,10 +871,8 @@ def _error_handler(self,response):
                 code = error.split("  ")[1]
                 message =  error.split("  ")[2]
 
-        if message:
-
-            sys.tracebacklimit = 0
-            raise DevoClientException("Query lauched reported the following error -----> " + message)
+        sys.tracebacklimit = 0
```

Comment:
I don't understand this limit very well. The more information we provide, the better.